crates/containerd-shim-wasm/src/sandbox/instance_utils.rs

```diff
@@ -1,10 +1,11 @@
 //! Common utilities for the containerd shims.
 use crate::sandbox::error::Error;
 use anyhow::{bail, Context, Result};
+use nix::unistd::{dup, dup2};
 use std::{
-    fs::{self, OpenOptions},
+    fs::{self, File, OpenOptions},
     io::ErrorKind,
-    os::fd::{IntoRawFd, RawFd},
+    os::fd::AsRawFd,
```

Comment:
**question:** Does this build on Windows? I thought `AsRawFd` was only available on unix.

Reply:
It is, will clean it up in #238